specification/wasm-3.0/6.3-text.modules.spectec

```diff
@@ -253,7 +256,7 @@ grammar Telemtable_(I)/abbrev : () hint(show Ttable_(%)) =
 ;; Modules
 
 syntax decl hint(name "declaration") =
-  type | import | tag | global | mem | table | func | data | elem | start | export
+  type | import* | tag | global | mem | table | func | data | elem | start | export
```

Comment:
You can't union types that aren't flat variants. But this also doesn't look right conceptually, an import should still be a decl individually.